packages/datadog_flutter_plugin/ios/datadog_flutter_plugin/Sources/datadog_flutter_plugin/DatadogSdkPlugin.swift

```diff
@@ -9,6 +9,29 @@ import DatadogCrashReporting
 import DatadogInternal
 import DatadogRUM
 import DatadogLogs
+// This is a bit of a hack required for both SPM and Cocoapods to play nice
+// with the C header (SPM requires it in a separate module, while Cocoapods does not.)
+#if canImport(datadog_flutter_plugin_c)
+    import datadog_flutter_plugin_c
+#endif
+
+@_cdecl("flutterGetDatadogContext")
+func flutterGetDatadogContext() -> DatadogCContext {
+    let context = ContextMessageReceiver.shared.cachedContext
+    guard let context = context else {
+        return DatadogCContext(sessionId: nil, accountId: nil, userId: nil)
+    }
+
+    let cSessionID = context.sessionId?.withCString { strdup($0) }
+    let cAccountId = context.accountId?.withCString { strdup($0) }
+    let cUserId = context.userId?.withCString { strdup($0) }
```

Comment:
These will leak if `flutterGetDatadogContext()` is called but the resulting `DatadogCContext` is not consumed, right? Who is freeing those strings?